server/static/js/index/home_page_modal_windows.js

```diff
@@ -0,0 +1,298 @@
+"use strict";
+// Variables with classes for buttons
+const OPEN_MODAL_USER = ".open-modal-user";
+const OPEN_MODAL_DEVICE = ".open-modal-device";
+const CLOSE_MODAL = ".close-modal";
+const TABLE_ROW = ".table-row";
+const BTN_MODAL_DEVICE_EDIT = '#btn_modal_devise_edit';
+const BTN_MODAL_DEVICE_ADD = '#btn_modal_device_add';
+const BTN_MODAL_USER_ADD = '#btn_modal_user_add';
+const BTN_MODAL_USER_EDIT = '#btn_modal_user_edit';
+//Other variables
+const MODAL = ".modal";
+const MODAL_ANIMATION_OPEN = "modal-animation-open";
+const MODAL_ANIMATION_CLOSE = "modal-animation-close";
+
+const MODAL_DEVICE = "#modal_device";
+const MODAL_DEVICE_ID_HIDDEN = "#modal_device_id_hidden";
+const MODAL_DEVICE_ID = "#modal_device_id";
+const MODAL_DEVICE_TYPE = "#modal_device_type";
+const MODAL_DEVICE_KEY = "#modal_device_key";
+
+const MODAL_USER = "#modal_user";
+const MODAL_USER_ID_HIDDEN = "#modal_device_id_hidden";
+const MODAL_USER_FIRST_NAME = "#modal_user_first_name";
+const MODAL_USER_LAST_NAME = "#modal_user_last_name";
+const MODAL_USER_EMAIL = "#modal_user_email";
+const MODAL_USER_PASSWORD = "#modal_user_password";
+const MODAL_USER_CONFIRM_PASSWORD = "#modal_user_confirm_password";
+const MODAL_USER_CHECKBOX = '.modal-user-checkbox';
+
+const DISPLAY_BLOCK = {"display":"block"};
+
+let rowSelected = false;
+let $activeRow;
+
+
+/**
+ * Resets table row event listener, allowing it to be selected/deselected
+ * @var $activeRow        - It contains selected row
+ * @var rowSelected       - Active row indicator
+ */
+function resetRowSelection () {
+    $(TABLE_ROW).click(function () {
+        $activeRow = $(this);
+        $(TABLE_ROW).removeClass("row-active");
+        $activeRow.addClass("row-active");
+        $('[data-modal="edit"]').addClass("btn-active");
+        $('[data-modal="debug"]').addClass("btn-active");
+        rowSelected = true;
+    });
+}
+
+/**
+ * The function sets the values of the modal window fields in accordance with the selected line (for devices)
+ */
+function setDeviceModalWindowValues () {
+    let row = $activeRow;
```

Comment:
Didn't notice it the first time - this variable is redundant, since you simply assign it to another, existing variable. Consider removing it.